.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,38 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+concurrency: test-${{ github.sha }}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      PYTHON_VERSION: "3.10"
+
+    steps:
+      - name: Checks out repository
+        uses: actions/checkout@v3
```

Comment:
```
  setup:
    runs-on: self-hosted
    steps:
    - uses: actions/checkout@v2
    - name: Set Up ENV
      run: |
          echo "${{ secrets.ASYNCMY_URI }}" > .env
          echo "${{ secrets.DISCORD_ASYNCMY_URI }}" >> .env
```

Reply:
This doesn't seem to work. Will try adding run as it's own key